grpc/pb/space.proto

```diff
@@ -14,12 +14,23 @@ service SpaceApi {
   rpc OpenFile(OpenFileRequest) returns (OpenFileResponse) {}
   rpc AddItems(AddItemsRequest) returns (stream AddItemsResponse) {}
   rpc CreateFolder(CreateFolderRequest) returns (CreateFolderResponse) {}
-}
 
-message ListDirectoriesRequest {
+  // Identities
+  rpc GetIdentityByUsername(GetIdentityByUsernameRequest) returns (GetIdentityByUsernameResponse) {}
+  rpc CreateUsernameAndEmail(CreateUsernameAndEmailRequest) returns (CreateUsernameAndEmailResponse) {}
```

Comment:
This endpoint would encapsulate two backend calls, one to create a username and one to create an email for that username. If email is not provided it will only do the first call